### PR TITLE
docs(FAQ): Remove expired links

### PR DESCRIPTION
**Pull request by selicens**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/react/faq.zh-CN.md [hunk 1]
```diff
@@ -105,21 +105,21 @@ antd 内部会对 props 进行浅比较实现性能优化。当状态变更，
...
-| Ant Design 2.x | https://ant-design-2x.gitee.io |
-| Ant Design 1.x | https://ant-design-1x.gitee.io |
-| Ant Design Pro | https://ant-design-pro.gitee.io/ |
-| Ant Design Mobile | https://ant-design-mobile.antgroup.com/zh <br /> https://antd-mobile.gitee.io/ |
-| Ant Design Mini | https://ant-design-mini.antgroup.com <br /> https://antd-mobile.gitee.io/ |
-| Ant Design Charts | https://ant-design-charts.antgroup.com<br /> https://antd-mobile.gitee.io/ |
+| Ant Design 3.x | https://3x.ant.design |
+| Ant Design 2.x | https://2x.ant.design |
+| Ant Design 1.x | https://1x.ant.design |
+| Ant Design Pro | https://pro.ant.design/zh-CN/ |
```
Comment (afc163) [inline docs/react/faq.zh-CN.md:117]: 不能留这些，这些不是国内镜像地址，得删掉。

Comment (selicens) [reply docs/react/faq.zh-CN.md:117]: 那现在有历史版本的国内镜像嘛

Comment (afc163) [reply docs/react/faq.zh-CN.md:117]: 现在应该都没了。

Comment (selicens) [reply docs/react/faq.zh-CN.md:117]: 1

Comment (selicens) [reply docs/react/faq.zh-CN.md]: > 现在应该都没了。

已修改

**Review by afc163 — DISMISSED** [review]

File: docs/react/faq.zh-CN.md [hunk 2]
```diff
@@ -105,21 +105,17 @@ antd 内部会对 props 进行浅比较实现性能优化。当状态变更，
...
-| Ant Design Pro | https://ant-design-pro.gitee.io/ |
-| Ant Design Mobile | https://ant-design-mobile.antgroup.com/zh <br /> https://antd-mobile.gitee.io/ |
-| Ant Design Mini | https://ant-design-mini.antgroup.com <br /> https://antd-mobile.gitee.io/ |
-| Ant Design Charts | https://ant-design-charts.antgroup.com<br /> https://antd-mobile.gitee.io/ |
+| Ant Design Mobile | https://ant-design-mobile.antgroup.com/zh |
+| Ant Design Mini | https://ant-design-mini.antgroup.com |
+| Ant Design Charts | https://ant-design-charts.antgroup.com |
 | AntV | https://antv.antgroup.com |
-| Ant Motion | https://ant-motion.gitee.io |
+| Ant Motion | https://motion.ant.design |
```
[the remark this answers is not in the document]
Comment (selicens) [reply docs/react/faq.zh-CN.md]: 1

**Review by afc163 — APPROVED** [review]

**Pull request merged**